src/printer.rs

```diff
@@ -4,11 +4,90 @@ use crate::histogram::histogram;
 use average::Max;
 use average::Variance;
 use byte_unit::Byte;
+use crossterm::style::{StyledContent, Stylize};
+use http::StatusCode;
 use std::collections::BTreeMap;
 use std::io::Write;
 use std::time::Duration;
 use std::time::Instant;
 
+#[derive(Clone, Copy)]
+struct StyleScheme {
+    color_enabled: bool,
+}
+impl StyleScheme {
+    fn no_color(self, text: String) -> StyledContent<String> {
```

Comment:
I think all `text: String` in `impl` can be rewritten as `text: &str` and a bunch of `.to_owned()` will disappear.

e.g `fn no_color(self, text: String) -> StyledContent<String>` to
`fn no_color(self, text: &str) -> StyledContent<&str>`